wcivf/apps/elections/templates/elections/election_view.html

```diff
@@ -85,7 +85,8 @@ <h3>{% blocktrans trimmed with title=object.pluralized_division_name|title %} {{
     {% include "elections/includes/_postcode_search_form.html" %}
 
     {% include "feedback/feedback_form.html" %}
-
-
+    <div id="report_form_container" style="display: none;">
+        {% include "bug_reports/report_form_inline.html" %}
+    </div>
```

Comment:
See what Peter thinks, but I don't think we want this on anything but the postcode and ballot views?